stories/molecules/Dropdown.stories.tsx

```diff
@@ -2,138 +2,83 @@ import * as React from 'react';
-        display: 'flex',
-    },
-    divider: {
-        borderBottom: '1px solid #80808045',
-    },
-    icon: {
-        marginRight: '10px',
-    },
-};
 
-const meta: Meta<typeof Dropdown> = {
+const meta: Meta<typeof DropdownContent> = {
     title: 'Molecules/Dropdown',
-    component: Dropdown,
+    component: DropdownContent,
 };
 
 export default meta;
 
-type Story = StoryObj<typeof Dropdown>;
+type Story = StoryObj<typeof DropdownContent>;
 
 export const _Dropdown: Story = {
-    name: 'Dropdown',
+    name: 'Actions Dropdown',
     args: {},
-    render: (args) => {
-        const onDropdownStateChange = (object) => {
-            console.log(
-                'onDropdownStateChange was called with the following:',
-                JSON.stringify(object)
-            );
-        };
-
-        const customButtonsDemo = [
-            <Button context="primary">Click me!</Button>,
-            <Button>
-                <HiDotsVertical />
-            </Button>,
-            <Button variant="ghost">Select any</Button>,
-        ];
-
-        return (
-            <div style={styles.content}>
-                <Dropdown<string>
-                    {...args}
-                    button={customButtonsDemo[0]}
-                    additionalSelectProps={{
-                        onStateChange: onDropdownStateChange,
-                    }}
-                >
-                    <ListItem key="disabled-key" disabled style={styles.divider}>
-                        Disabled ListItem
-                    </ListItem>
-                    <Separator title="Section title" />
-                    <ListItem key="first-key" value="first-value">
-                        ListItem with value
-                    </ListItem>
-                    <Separator />
-                    {['ListItem with value 1', 'ListItem with value 2'].map((value) => (
-                        <ListItem key={value} value={value}>
-                            {value}
-                        </ListItem>
-                    ))}
-
-                    <ListItem key="second-key" value="second-value">
-                        <div style={styles.customListItem}>
-                            <IconTextkernel style={styles.icon} />
-                            <strong>Custom ListItem with value</strong>
-                        </div>
-                    </ListItem>
-                    <div style={styles.customDiv}>Just custom div element</div>
-                </Dropdown>
-            </div>
-        );
-    },
+    render: () => <p>To be implements</p>,
 };
 
 export const _DropdownWithSingleSelectItem: Story = {
     name: 'Dropdown - Single Select Item',
     args: {},
     render: (args) => {
         const [selectedValue, setSelectedValue] = React.useState<string>();
-
-        const onDropdownStateChange = (object) => {
-            console.log(
-                'onDropdownStateChange was called with the following:',
-                JSON.stringify(object)
```

Comment:
let’s keep the `console.log` to track state changes, so it stays consistent with all other stories, what do you think?

Reply:
@AnastasiiaMP what type of `state changes` do you mean? Do we have props for that?

Reply:
mostly onChange/onSelect events which changes component’s internal state, such as input values or selected items. Basically as we do in all stories, I am referring to `onDropdownStateChange` here which was removed